Springboot-demo/src/main/java/com/hollszabolcs/demo/repository/EmployeeRepository.java

```diff
@@ -0,0 +1,7 @@
+package com.hollszabolcs.demo.repository;
+
+import com.hollszabolcs.demo.model.Employee;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface EmployeeRepository extends JpaRepository<Employee, Long> {
```

Comment:
The id is not a long here, but you are referring here as a long - I am not sure if this project even runs properly